api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -88,3 +90,26 @@ message HttpService {
   // authorization server. Note that these will override the headers coming from the downstream.
   repeated envoy.api.v2.core.HeaderValue authorization_headers_to_add = 6;
 }
+
+// Extra settings on a per virtualhost/route/weighter-cluster level.
+message ExtAuthzPerRoute {
+  oneof override {
+    option (validate.required) = true;
+
+    // Disable the ext auth filter for this particular vhost or route.
+    bool disabled = 1 [(validate.rules).bool.const = true];
```

Comment:
Why do you have this field if this constraint is set?

Reply:
I copied the pattern from the buffer filter. 
I believe the assumption is that you chose the disable in one_of then you probably want it set to true.

Reply:
Ah, right, makes sense.